client/my-sites/plan-features/index.jsx

```diff
@@ -200,6 +200,7 @@ class PlanFeatures extends Component {
 			return (
 				<div className="plan-features__mobile-plan" key={ planName }>
 					<PlanFeaturesHeader
+						available={ available }
```

Comment:
I find the `available` prop a bit ambiguous by itself. Disregarding the fact that we already have a bunch of boolean props in this section of the code (a big bunch that we should start reducing IMHO), the prop itself feels unclear - reading the code I'm not sure which of the following it represents:

* Plan is available (or not)
* Plan feature is available (or not)
* Plan features are available (or not)
* Plan features header is available (or not)

Reply:
@tyxla I derived this name from other components used on plans page, such as , but to make the meaning clearer I just renamed it to `availableForPurchase` - does that work better for you?

Reply:
Oh well, thanks for updating this, but changing only instances that this PR touches and not updating the others doesn't make it more readable 😕